theme.json

```diff
@@ -1,75 +1,165 @@
+            "title": "Footer Container",
+            "name": "footer",
+            "area": "footer"
+        }
+    ],
+    "styles": {
+        "spacing": {
+            "padding": {
+                "top": "0px",
+                "right": "2rem",
+                "bottom": "0px",
+                "left": "2rem"
+            }
         },
-        {
-          "slug": "foreground",
-          "color": "var(--wp--preset--color--typography-primary-color)",
-          "name": "Foreground"
+        "elements": {
+            "h6": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h6)"
+                }
+            },
+            "h5": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h5)"
+                }
+            },
+            "h4": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h4)"
+                }
+            },
+            "h3": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h3)"
+                }
+            },
+            "h2": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h2)"
+                }
+            },
+
+            "h1": {
+                "typography": {
+                    "fontSize": "var(--dswp-typography-font-size-h1)"
+                }
+            }
+        }
+    },
+    "settings": {
+        "appearanceTools": true,
+        "useRootPaddingAwareAlignments": true,
+        "layout": {
+            "contentSize": "1440px",
+            "wideSize": "1440px"
         },
-        {
-          "slug": "background",
-          "color": "#ffffff",
-          "name": "Background"
+        "typography": {
+            "fontSizes": [
+                {
+                    "slug": "bcds-h6",
+                    "size": "var(--dswp-typography-font-size-h6)",
+                    "name": "H6"
+                },
+                {
+                    "slug": "bcds-h5",
+                    "size": "var(--dswp-typography-font-size-h5)",
+                    "name": "H5"
+                },
+                {
+                    "slug": "bcds-h4",
+                    "size": "var(--dswp-typography-font-size-h4)",
+                    "name": "H4"
+                },
+                {
+                    "slug": "bcds-h3",
+                    "size": "var(--dswp-typography-font-size-h3)",
+                    "name": "H3"
+                },
+                {
+                    "slug": "bcds-h2",
+                    "size": "var(--dswp-typography-font-size-h2)",
+                    "name": "H2"
+                },
+                {
+                    "slug": "bcds-h1",
+                    "size": "var(--dswp-typography-font-size-h1)",
+                    "name": "H1"
+                },
+                {
```

Comment:
These sizes from here to `bcds-large-body` are all smaller than h6 so the list isn't in order of smallest to largest.

Reply:
fixing in next PR